tests/jsUnit/global/navbar.spec.js

```diff
@@ -0,0 +1,51 @@
+import navbar from '../../../resources/js/globals/navbar'
```

Comment:
Make sure the paths are relative ('./../' instead of '../../)

Reply:
👍